### DISCUSSION TITLE
How to install detectron2 in pytorch1.8

### DISCUSSION THREAD

**Discussion by linchundan88** [facebookresearch/detectron2#2701] [Q&A]
print(torch.version.cuda)  displays 11.1.
However, when I run the command "python -m pip install -e detectron2",  I got the following error:
    error: command '/usr/local/cuda-11.0/bin/nvcc' failed with exit status 1.

  cuda version? I do not know.

**Comment by ppwwyyxx** [comment]
If you need help to solve an unexpected issue you observed, please include details following the ["Unexpected behaviors" issue template](https://github.com/facebookresearch/detectron2/issues/new/choose).
